internal/pkg/instrumentation/bpf/google.golang.org/grpc/server/probe.go

```diff
@@ -141,11 +165,36 @@ func (w writeStatusConst) InjectOption(td *process.TargetDetails) (inject.Option
 	return inject.WithKeyValue("write_status_supported", writeStatus), nil
 }
 
+type serverAddrConst struct{}
+
+var (
+	serverAddrMinVersion = version.Must(version.NewVersion("1.60.0"))
+	serverAddr           = false
+)
+
+func (w serverAddrConst) InjectOption(td *process.TargetDetails) (inject.Option, error) {
+	ver, ok := td.Libraries[pkg]
+	if !ok {
+		return nil, fmt.Errorf("unknown module version: %s", pkg)
+	}
+	if ver.GreaterThanOrEqual(serverAddrMinVersion) {
+		serverAddr = true
+	}
+	return inject.WithKeyValue("server_addr_supported", serverAddr), nil
+}
+
 // event represents an event in the gRPC server during a gRPC request.
 type event struct {
 	context.BaseSpanProperties
 	Method     [100]byte
 	StatusCode int32
+	LocalAddr  NetAddr
+}
+
+type NetAddr struct {
+	IP   [16]uint8
+	Port uint64
+	IsV6 uint64
```

Comment:
This doesn't ever seem to be used. Should it be removed?

Reply:
Yup think we can remove it